docs/conf.py

```diff
@@ -5,18 +5,17 @@
 
 project = "NeuroConv"
 copyright = "2022, CatalystNeuro"
-author = "Cody Baker and Ben Dichter"
+author = "Cody Baker, Heberto Mayorquin, Szonja Weigl and Ben Dichter"
 
 extensions = [
-    "sphinx.ext.autodoc",
-    "sphinx.ext.autosummary",
-    "sphinx.ext.napoleon",
-    "myst_parser",
-    "sphinx_toggleprompt",
-    "sphinx_copybutton",
-    "sphinx.ext.intersphinx",
-    "sphinx_search.extension",
-    "sphinx.ext.viewcode",
+    "sphinx.ext.napoleon",  # Support for NumPy and Google style docstrings
+    "sphinx.ext.autodoc",  # Includes documentation from docstrings in docs/api
+    "sphinx.ext.autosummary", # To-add
+    "sphinx_toggleprompt",  # Used to control >>> behavior in the conversion gallery example doctests
+    "sphinx_copybutton",  # Used to control the copy button behavior in the conversion gallery doctsts
+    "sphinx.ext.intersphinx",  # Allows links to other sphinx project documentation sites
+    "sphinx_search.extension",  # Alows for auto search function the documentation
+    "sphinx.ext.viewcode",  # Shows source code in the documentation
```

Comment:
I really like these comments @h-mayorquin!

Reply:
Every time that I encounter a documentation error I am wondering if any of the extensions is interfering with another but I fail to recall what they do. Hopefully this will help us in the future!